lib/Steps/Integrations/NextJs.ts

```diff
@@ -97,6 +94,59 @@ export class NextJs extends BaseIntegration {
     return this.shouldConfigure;
   }
 
+  private _createSentryCliConfig(cliProps: SentryCliProps): void {
+    const authToken = cliProps['auth/token'];
+
+    /**
+     * To not commit the auth token to the VCS, instead of adding it to the
+     * properties file (like the rest of props), it's added to the Sentry CLI
+     * config, which is added to the gitignore. This way makes the properties
+     * file safe to commit without exposing any auth tokens.
+     */
+    if (authToken) {
+      const authTokenProp = { 'auth/token': authToken };
+      fs.appendFileSync(
+        SENTRYCLIRC_FILENAME,
+        this._sentryCli.dumpProperties(authTokenProp),
+      );
+      this._gitIgnoreFile(
+        SENTRYCLIRC_FILENAME,
+        `⚠ Could not add ${SENTRYCLIRC_FILENAME} to ${GITIGNORE_FILENAME}, ` +
+          'please add it to not commit your auth key.',
+      );
+      delete cliProps['auth/token'];
```

Comment:
Not a fan of mutating the function argument. Can we do this in an immutable way?